enable reload auditd rules (#10835)
elastic/beats: auditbeat/docs/modules/auditd.asciidoc

  include_raw_message: false
  include_warnings: false
  backpressure_strategy: auto
  reload:
    period: 30s

>     enabled: true

Reviewer: can you update this to correspond to the new default?

Author: 👍

*`backpressure_strategy`*:: Specifies the strategy that {beatname_uc} uses to
prevent backpressure from propagating to the kernel and impacting audited
processes.

> *`reload`*:: `enabled` auditbeat check auditd rule files in fixed interval 

Reviewer: This looks like it's in the middle of the `backpressure_strategy` documentation. Think it needs to go below the `The possible values are` block.

- `none`: No backpressure mitigation measures are enabled.
--

*`reload`*:: `enabled` auditbeat check auditd rule files in fixed interval 
`period`, reload auditd rule on host if any changes to the rule file detected.
Its similar to the config reload, but instead of .yml files it is for .rules

> files.

Reviewer: Same as above, these changes should go in: `auditbeat/module/auditd/_meta/docs.asciidoc`. In addition, after both this and the above change are made, you'll need to run `mage update` in the auditbeat directory

> *`reload`*:: `enabled` auditbeat check auditd rule files in fixed interval 
> `period`, reload auditd rule on host if any changes to the rule file detected.
> Its similar to the config reload, but instead of .yml files it is for .rules
> files.

Reviewer: ```suggestion
*`reload`*:: `enabled` Causes auditbeat to reload auditd rule files at a fixed interval if any changes to the rule files are detected.
`period`, Specifies the interval after which auditbeat will check and reload rule files.
```
